openpype/hosts/maya/plugins/publish/extract_thumbnail.py

```diff
@@ -27,21 +29,48 @@ def process(self, instance):
 
         camera = instance.data["review_camera"]
 
-        maya_setting = instance.context.data["project_settings"]["maya"]
-        plugin_setting = maya_setting["publish"]["ExtractPlayblast"]
-        capture_preset = plugin_setting["capture_preset"]
-        override_viewport_options = (
-            capture_preset["Viewport Options"]["override_viewport_options"]
-        )
+        host_name = instance.context.data["hostName"]
+        family = instance.data["family"]
+        task_data = instance.data["anatomyData"].get("task", {})
+        task_name = task_data.get("name")
+        task_type = task_data.get("type")
+        subset = instance.data["subset"]
+
+        filtering_criteria = {
+            "hosts": host_name,
+            "families": family,
+            "task_names": task_name,
+            "task_types": task_type,
+            "subset": subset
+        }
 
+        maya_settings = instance.context.data["project_settings"]["maya"]
+        plugin_settings = maya_settings["publish"]["ExtractPlayblast"]
+
+        capture_preset = plugin_settings["capture_preset"]
+        preset = {}
```

Comment:
This is the _deprecated behavior_, right? Let's comment that this is deprecated fallback.
I feel like this should be in the `else` for when no profiles were found below?

Reply:
Yeah, good point. Have added a comment to both extractors about deprecated settings.